pubspec.yaml

```diff
@@ -39,7 +39,8 @@ dependencies:
   font_awesome_flutter: 8.7.0
   # check internet connection availability
   data_connection_checker: ^0.3.4
-
+  get_it: 
```

Comment:
You are adding another lib for the same purpose. The only difference is: it does not require context. Does it really important?

Reply:
If you can propose another way to separate logic of the UI from the business logic of the App please propose. The logic of the app should not be dependent on current context. We are trying to achive different layer separation between Domain, Data and presentation layers. This separation is very important for future theming and other customization aspects like adding third-party web solutions (shopify for instance).